src/server/api/routers/form-response.ts

```
@@ -48,92 +54,260 @@ export const formResponseRouter = createTRPCRouter({
+          priority: z.enum(["ASC", "DESC"]).optional(),
+          userIds: z.array(z.string()).optional(),
+          setores: z.array(z.string()).optional(),
+          hasResponse: z.boolean().optional(),
+        })
+        .optional(),
+    )
+    .query(async ({ ctx, input }) => {
+      const currentUserId = ctx.auth.userId
+
+      // Construir where clause
+      const where: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)